Octokit/Models/Response/CommitCommentReaction.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Octokit.Internal;
+using System;
+using System.Diagnostics;
+using System.Globalization;
+
+namespace Octokit
+{
+    public enum Reaction
```

Comment:
I'd suggest the enum be named `ReactionType` since below ive suggested the actual response class is called `Reaction`